extensions/eclipse-che-theia-plugin-ext/src/node/che-api-service.ts

```diff
@@ -295,6 +295,15 @@ export class CheApiServiceImpl implements CheApiService {
         }
     }
 
+    async getOAuthProviders(): Promise<string[]> {
+        const cheApiClient = await this.getCheApiClient();
+        try {
+            return await cheApiClient.getOAuthProviders();
```

Comment:
PR check fails here because of error:
```
task-plugin: $ rimraf lib
@eclipse-che/theia-plugin-ext: [compile] src/node/che-api-service.ts(301,39): error TS2339: Property 'getOAuthProviders' does not exist on type 'IRemoteAPI'.
```